src/js/media-time-display.js

```diff
@@ -71,9 +74,39 @@ class MediaTimeDisplay extends MediaTextDisplay {
 
     this.setAttribute('role', 'progressbar');
     this.setAttribute('aria-label', nouns.PLAYBACK_TIME());
+
+    const keyUpHandler = (evt) => {
```

Comment:
seems like some boilerplate that I had to add here to handle keypresses, is there a more generic util for this?

Reply:
it's fine here I think, copy a little code